eventlog/messages.go

```diff
@@ -44,23 +43,39 @@ type InitResponse struct {
 	ID skipchain.SkipBlockID
 }
 
-// LogRequest is puts one or more new log events into the OmniLedger.
+// LogRequest puts one or more new log events into the OmniLedger.
 type LogRequest struct {
 	SkipchainID skipchain.SkipBlockID
 	Transaction omniledger.ClientTransaction
 }
 
+// LogResponse is the reply to LogRequest.
+type LogResponse struct {
+}
+
+// SearchRequest includes all the search parameters (AND of all provided search parameters).
+type SearchRequest struct {
+	ID    skipchain.SkipBlockID
+	Topic string // Return events where Event.Topic == Topic.
+	From  int64  // Return events where When is > From.
+	To    int64  // Retursn events where When is <= To.
```

Comment:
`s/Retursn/Returns/`

I would copy/paste the same documentation to the client-API, as there is the most probably place a user will look.